test/test_opportunity/test_opportunity.cpp

```diff
@@ -1,18 +1,24 @@
 #include "../test_util.hpp"
 #include <unity.h>
+#include <time.h>
+#include <stdio.h>
 
 extern "C"
 {
 #include <opportunity.h>
 }
 
+#define RANDOM_SEED time(0)
+#define INITIALIZE_SEED(x) srand(x)
```

Comment:
Can we just have one function? There's no difference between initializing and reseting the seed